CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## 0.26.0 / 2024-05-08
+
+* [CHANGE] Improve generator parse error handling #1167
+
```

Comment:
```suggestion
```

Reply:
removed line in commit 03a69f9d75d0b9e40a786dc8e60f97d51ffcbb97